### PR TITLE
Add dnscrypt-wrapper v0.1.15 (new formula).

### PR DESCRIPTION
**Pull request by Cofyc**
This is dnscrypt wrapper, server-side dnscrypt proxy.

### CODE REVIEW

File: Library/Formula/dnscrypt-wrapper.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  def install
+    args = ["--prefix=#{prefix}"]
+
+    system "make configure"
+    system "./configure", *args
+    system "make"
+    system "make install"
+  end
+end
```
Comment (mikemcquaid) [inline Library/Formula/dnscrypt-wrapper.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/dnscrypt-wrapper.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require "formula"
```
Comment (mikemcquaid) [inline Library/Formula/dnscrypt-wrapper.rb]: Remove this line.

File: Library/Formula/dnscrypt-wrapper.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  url "https://github.com/Cofyc/dnscrypt-wrapper/releases/download/v0.1.15/dnscrypt-wrapper-v0.1.15.tar.bz2"
+  sha256 "d486f3f923c6809c830e9db39290b0f44b1683f63f8bd3aeaa6225c64af232c1"
+
+  depends_on "autoconf" => :build
+
+  depends_on "libsodium"
+  depends_on "libevent"
+
+  def install
+    args = ["--prefix=#{prefix}"]
```
Comment (mikemcquaid) [inline Library/Formula/dnscrypt-wrapper.rb]: Just inline these args as they don't vary.

File: Library/Formula/dnscrypt-wrapper.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+    system "make configure"
```
Comment (mikemcquaid) [inline Library/Formula/dnscrypt-wrapper.rb]: `system "make", "configure"`

File: Library/Formula/dnscrypt-wrapper.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    system "./configure", *args
+    system "make"
+    system "make install"
```
Comment (mikemcquaid) [inline Library/Formula/dnscrypt-wrapper.rb]: `system "make", "install"`

File: Library/Formula/dnscrypt-wrapper.rb [hunk 6]
```diff
@@ -0,0 +1,22 @@
+require "formula"
+
+class DnscryptWrapper < Formula
+  homepage "http://cofyc.github.io/dnscrypt-wrapper/"
+  head "https://github.com/Cofyc/dnscrypt-wrapper.git"
```
Comment (mikemcquaid) [inline Library/Formula/dnscrypt-wrapper.rb]: Put `head` below `sha256`.

File: Library/Formula/dnscrypt-wrapper.rb (hunk 1 above)
Comment (Cofyc) [inline Library/Formula/dnscrypt-wrapper.rb]: @mikemcquaid Style problems are fixed. Added a few test commands, but currently it doesn't have comprehensive test.

**Comment by mikemcquaid** [comment]
Great work, thanks!

**Pull request closed without merging**